edenai_apis/apis/xai/xai_llm_api.py

```
@@ -0,0 +1,86 @@
+
+import httpx
+from openai import BaseModel
+from features.llm.llm_interface import LlmInterface
+from edenai_apis.features.llm.chat.chat_dataclass import ChatCompletionResponse
+
+
+class XAiLLMApi(LlmInterface):
+
+    def llm__chat(
+        self,
+        messages: List = [],
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)